solutions/fully-configurable/main.tf

```diff
@@ -131,6 +139,55 @@ module "buckets" {
   ]
 }
 
+data "ibm_iam_account_settings" "iam_account_settings" {
```

Comment:
Please avoid using this data lookup as it requires you to be an account admin. Can you get the account ID from the bucket CRN perhaps?

Reply:
oh right yes true - I guess we can leave it for now, but ensure that:
a) add a count around it as its only needed if doing cross account auth policy
b) you leave a code comment explaining why its needed
c) ensure all documented permissions are updated to include this